scripts/lisk_bridge.sh

```diff
@@ -48,7 +51,7 @@ extractConfig() {
 	export PORT
 	PORT="$(grep "port" "$LISK_CONFIG" | head -1 | cut -d':' -f 2 | cut -d',' -f 1 | tr -d '[:space:]')"
 
-	readarray secrets < <(jq -r '.forging.secret | .[]' "$LISK_CONFIG")
+	readarray secrets < <($JQ -r '.forging.secret | .[]' "$LISK_CONFIG")
```

Comment:
Should be double-quoted. Did you run shellcheck?

Reply:
yeah, it didn't yell at me -- I'll add them though

Reply:
Something must be wrong with shellcheck on Jenkins thenm, it's reporting 1 warning and 8 info messages on my machine.